go.mod

```diff
@@ -108,10 +108,9 @@ require (
 	golang.org/x/net v0.1.0 // indirect
 	golang.org/x/oauth2 v0.0.0-20210402161424-2e8d93401602 // indirect
 	golang.org/x/sys v0.1.0 // indirect
-	golang.org/x/text v0.4.0 // indirect
```

Comment:
why are we changing this?

Reply:
It's just from a module update I needed to do as part of upgrading to go1.19 locally. Looks like there were a few minor changes between now and the last module update, so I figured there was no harm in keeping them in this change.